src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -2189,5 +2189,40 @@ namespace wallet_rpc
     };
   };
 
+  struct COMMAND_RPC_VALIDATE_ADDRESS
+  {
+    struct request_t
+    {
+      std::string address;
+      bool any_net_type;
+      bool allow_openalias;
+
+      BEGIN_KV_SERIALIZE_MAP()
+        KV_SERIALIZE(address)
+        KV_SERIALIZE_OPT(any_net_type, false)
+        KV_SERIALIZE_OPT(allow_openalias, false)
+      END_KV_SERIALIZE_MAP()
+    };
+    typedef epee::misc_utils::struct_init<request_t> request;
```

Comment:
Looks like this class template isn't found anywhere?
```
/home/buildbot/builder/monero-static-ubuntu-amd64/build/src/wallet/wallet_rpc_server_commands_defs.h:2206:31: error: 'struct_init' in namespace 'epee::misc_utils' does not name a template type
     typedef epee::misc_utils::struct_init<request_t> request;
                               ^~~~~~~~~~~
```

Reply:
Looks like it's in #5101

Reply:
Builds fine when rebased on #5101. Note that there is a small merge conflict in p2p/net_node.h